frontend/src/components/DetailsPanel/index.js

```diff
@@ -131,6 +133,34 @@ function DetailsPanel({ classId, classList, setClassId, children, dynamicShiftbu
     }
   };
 
+  const renderInstructorInfo = () => {
+    if (!shiftDetails || !shiftDetails.shift_type || !panelInfo?.instructor_email) return;
+  
+    return (
+      <>
+        {panelInfo?.instructor_f_name && panelInfo?.instructor_l_name
+          ? `${panelInfo.instructor_f_name} ${panelInfo.instructor_l_name}`
+          : "No instructor available"}
+        {shiftDetails.shift_type === "my-shifts" 
```

Comment:
the shift types are constants in frontend/src/data/constants.js. Instead of checking the type here, lets import the constant and use it here